### ISSUE TITLE
Block comments not preserved correctly

### ISSUE THREAD

**Issue by adamreisnz** [gazugafan/vscode-indent-on-paste#16]
Thanks for this great plugin. 

I noticed a problem with block comments that get deformed after pasting when using this plugin:

```js
/**
 * Local event check
 */
EventSchema
  .virtual('isLocal');
```

Copy pasting this yields:

```js
/**
* Local event check
*/
EventSchema
  .virtual('isLocal')
```

Note how the extra space in the comment block is swallowed.

This is with Javascript, indent on paste v 2.3.5

**Comment by gazugafan** [comment]
Thanks for the report. Should be fixed in v2.4.0 just released! Only full indents will be stripped now.

**Comment by adamreisnz** [comment]
Thanks, will check it out!
